### PR TITLE
Add licensing error codes

### PR DESCRIPTION
**Pull request by eeff**

### CODE REVIEW

File: include/neu_errcodes.h [hunk 1]
```diff
@@ -94,6 +94,13 @@ typedef enum {
     NEU_ERR_LIBRARY_INFO_INVALID  = 2302,
     NEU_ERR_LIBRARY_NAME_CONFLICT = 2303,
 
+    NEU_ERR_LICENSE_NOT_FOUND = 2400,
+    NEU_ERR_LICENSE_INVALID   = 2401,
+    NEU_ERR_LICENSE_EXPIRED   = 2402,
+    NEU_ERR_LICENSE_DISABLED  = 2403,
```
Comment (fengzeroz) [inline include/neu_errcodes.h:100]: what different with invalid

File: plugins/restful/http.c [hunk 2]
```diff
@@ -262,6 +262,12 @@ int http_response(nng_aio *aio, neu_err_code_e code, char *content)
     case NEU_ERR_PLUGIN_TAG_NOT_ALLOW_WRITE:
     case NEU_ERR_PLUGIN_TAG_NOT_EXIST:
     case NEU_ERR_PLUGIN_GRP_NOT_SUBSCRIBE:
+    case NEU_ERR_LICENSE_NOT_FOUND:
```
Comment (fengzeroz) [inline plugins/restful/http.c:265]: 404

File: include/neu_errcodes.h (hunk 1 above)
Comment (eeff) [reply include/neu_errcodes.h:100]: invalid for indicating the license is invalid in format,
disabled is for indicating a certain plugin is disabled/enabled.

File: plugins/restful/http.c (hunk 2 above)
Comment (eeff) [reply plugins/restful/http.c]: Fix

File: plugins/restful/http.c [hunk 3]
```diff
@@ -262,6 +262,12 @@ int http_response(nng_aio *aio, neu_err_code_e code, char *content)
...
+    case NEU_ERR_LICENSE_INVALID:
```
[the remark this answers is not in the document]
Comment (eeff) [reply plugins/restful/http.c]: Fix.

**Pull request merged**